src/blocks/content-helper/current-post-details/component.tsx

```diff
@@ -173,8 +184,8 @@ function ReferrerTypesSection( data: PostPerformanceData ) {
 					return (
 						<div className={ 'bar-fill ' + key } key={ key }
 							title={
-								/* translators: %s: Percentage value, %%: Escaped percent sign */
-								sprintf( __( '%s%%', 'wp-parsely' ), value.viewsPercentage ) // eslint-disable-line @wordpress/valid-sprintf
+								/* translators: 1: Referrer type, 2: Percentage value, %%: Escaped percent sign */
+								sprintf( __( '%s: %s%%', 'wp-parsely' ), getKeyTitle( key ), value.viewsPercentage ) // eslint-disable-line @wordpress/valid-sprintf
```

Comment:
Strings with multiple placeholders should use numbered placeholders.
```suggestion
								sprintf( __( '%1$s: %2$s%%', 'wp-parsely' ), getKeyTitle( key ), value.viewsPercentage ) // eslint-disable-line @wordpress/valid-sprintf
```